### PR TITLE
add python3-qrcode key to python.yaml

### PR DESCRIPTION
**Pull request by p-panagiotidis**
Please add the following dependency to the rosdep database.

## Package name:

python3-qrcode

## Package Upstream Source:

https://github.com/lincolnloop/python-qrcode

## Purpose of using this:

Python library for generating QR codes

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/bullseye/python3-flask-sqlalchemy
- Ubuntu: https://packages.ubuntu.com/
   - https://packages.ubuntu.com/focal/python3-qrcode
- Fedora: https://packages.fedoraproject.org/
  - https://packages.fedoraproject.org/pkgs/python-qrcode/python3-qrcode/
- Arch: https://www.archlinux.org/packages/
  - https://archlinux.org/packages/community/any/python-qrcode/
- Gentoo: https://packages.gentoo.org/
  - https://packages.gentoo.org/packages/dev-python/qrcode
- NixOS/nixpkgs: https://search.nixos.org/packages
  - https://search.nixos.org/packages?channel=22.11&show=python39Packages.qrcode&from=0&size=50&buckets=%7B%22package_attr_set%22%3A%5B%22python39Packages%22%5D%2C%22package_license_set%22%3A%5B%5D%2C%22package_maintainers_set%22%3A%5B%5D%2C%22package_platforms%22%3A%5B%5D%7D&sort=relevance&type=packages&query=qrcode

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -8726,6 +8726,13 @@ python3-pyxdg-pip:
   ubuntu:
     pip:
       packages: [pyxdg]
+python3-qrcode:
+  arch: [python-qrcode]
+  debian: [python3-qrcode]
+  fedora: [python3-qrcode]
+  gentoo: [dev-python/qrcode]
+  nixos: [python39Packages.qrcode]
+  ubuntu: [python3-qrcode]
```
Comment (nuclearsandwich) [inline rosdep/python.yaml:8735]: Consider adding additional definitions for opensuse and rhel as suggested by the linter.

```suggestion
python3-qrcode:
  arch: [python-qrcode]
  debian: [python3-qrcode]
  fedora: [python3-qrcode]
  gentoo: [dev-python/qrcode]
  nixos: [python39Packages.qrcode]
  opensuse: [python3-qrcode]
  rhel: [python3-qrcode]
  ubuntu: [python3-qrcode]
```

**Review by nuclearsandwich — COMMENTED** [review]
I don't see any issues as-is but there are some improvements suggested by the repo checker.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -8726,6 +8726,16 @@ python3-pyxdg-pip:
...
       packages: [pyxdg]
+python3-qrcode:
+  arch: [python-qrcode]
+  debian: [python3-qrcode]
+  fedora: [python3-qrcode]
+  gentoo: [dev-python/qrcode]
+  nixos: [python39Packages.qrcode]
+  opensuse: [python3-qrcode]
+  rhel:
+    '8': [python3-qrcode]
```
Comment (ppanagiotidis) [inline rosdep/python.yaml:8737]: python3-qrcode was not available for RHEL 7

Comment (hidmic) [reply rosdep/python.yaml:8737]: I don't see it either.

**Review by hidmic — APPROVED** [review]
LGTM

**Review by audrow — APPROVED** [review]

**Pull request merged**